sources/metrics/open measurement/OpenMeasurementController.swift

```diff
@@ -30,16 +32,23 @@ extension OpenMeasurement {
             if case .finished = openMeasurement {
                 return dispatcher(PlayerCore.openMeasurementDeactivated())
             }
-            guard case .loading(let adVerifications) = openMeasurement, adSession == nil else { return }
+            
+            if case .readyForLoad(let adVerifications) = openMeasurement {
+                self.adVerifications = adVerifications
+                return
+            }
+            
+            guard case .loading = openMeasurement, adSession == nil else { return }
             
             guard let serviceScript = serviceScript else {
                 dispatcher(PlayerCore.failedOMConfiguration(with: OpenMeasurement.Errors.scriptNotAvailable))
                 return
             }
-            guard let adView = adViewAction() else {
-                dispatcher(PlayerCore.failedOMConfiguration(with: OpenMeasurement.Errors.failedToGetAdView))
+            guard let adView = adViewAction(),
+                  let adVerifications = adVerifications else {
                 return
             }
+            
             do {
                 let input = OpenMeasurement.Input(partnerBundleName: "Oath2",
```

Comment:
I guess this should also be `Verizonmedia`

Reply:
Good point.